### ISSUE TITLE
modal are not showing when i click show modal  💀

### ISSUE THREAD

**Issue by sanba-anass** [skeletonlabs/skeleton#1754]
### Current Behavior

https://www.skeleton.dev/utilities/modals#:~:text=Demo-,Show%20Modal,-This%20feature%20uses

**Comment by endigo9740** [comment]
@sanba-anass can you confirm what steps you've implemented from our instructions here:

https://www.skeleton.dev/utilities/modals

We had someone else having this issue recently and they forgot to implement the `<Modal />` component in their root layout. Maybe check that?

If you're getting any console errors post that. Otherwise there's not enough information here for us to help you.

**Comment by chanmathew** [comment]
I was running into the same issue until I realized I had to add the <Modal /> component in the root layout.

As a suggestion I think it's not very clear in the documentation specifically for the "Direct method" section, it doesn't tell you to add the component to the root layout. Perhaps that can be revised?

```
For one-off components you can create a ModalComponent object containing your component, props, and slot values.
```

**Comment by endigo9740** [comment]
It's there, but we will likely need to make it a bit more obvious. First example at the top

![image](https://github.com/skeletonlabs/skeleton/assets/1509726/59be653a-2d08-4f9a-a731-3bbedf773bb6)

**Comment by chanmathew** [comment]
Yeah i think the confusion is that I thought they are 3 different ways of implementing, and the "Direct method" where I'm trying to trigger my own component i thought was limited to the instructions only within that tab.

**Comment by endigo9740** [comment]
Understood. A few folks have missed this so we'll review how to make this more clear.
